documentation/local-dev/roslyn/readme.md

```diff
@@ -0,0 +1,37 @@
+# Instructions for Roslyn developers
+
+1. Check out a working copy of dotnet/roslyn, e.g. to the directory next to this repo's root `..\roslyn`.
+   Make sure the current working directory is the checkout of this repo.
+
+2. Copy LocalDev.Build.props to the root and set `RoslynRoot` property in the copied file to local Roslyn repo.
+
+    `cp documentation\local-dev\roslyn\LocalDev.Build.props LocalDev.Build.props`
+
+3. Copy LocalDev.Packages.props.roslyn to the root
+
+    `cp documentation\local-dev\roslyn\LocalDev.Packages.props LocalDev.Packages.props`
+
+4. Build Roslyn with:
+
+    `..\roslyn\Build.cmd -restore -pack`
+
+5. Clear the customized packages cache in `.packages`
+
+    `git clean -dxf .packages/`
```

Comment:
Why are you using `git clean` and not simply `rm -r`?

Reply:
Either way works. I just delete the directory from file explorer. @sharwell authored the original doc.